### PR TITLE
Documentation reorg

### PR DESCRIPTION
**Pull request by villares**
- Almost nothing left on the homepage
- Documenting differences #23 and main issues (like #17)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,53 +21,12 @@ def draw():
 
 start_p5(setup, draw)
 ```
+### [Documentation](https://github.com/berinhard/pyp5js/blob/master/docs/index.md)
```
Comment (berinhard) [inline README.md]: Should change the link to point to https://berinhard.github.io/pyp5js/

File: README.md [hunk 2]
```diff
@@ -22,53 +21,12 @@ def draw():
...
-
-After updating your code, you'll have to run the `transcrypt` command to update the files. Run it as:
-
-```
-$ pyp5js transcrypt my_sketch
-```
+- [Examples](https://berinhard.github.io/pyp5js/examples/)
 
-The way the code is implemented, for now, you'll have to execute it from the same directory-level where `my_sketch` is - you can't run it from inside the `my_sketch` directory.
+#### [Installation](https://github.com/berinhard/pyp5js/blob/master/docs/index.md#installation)
```
Comment (berinhard) [inline README.md]: Change to https://berinhard.github.io/pyp5js/#installation

File: README.md [hunk 3]
```diff
@@ -22,53 +21,12 @@ def draw():
...
 
-If you're lazy as me, you can use the `monitor` command instead of the previous one. The command will monitor your sketch directory and keep track of any changes on any `.py` files. When it notices a new change, it automatically runs the transcrypt process for you. So, now you'll just have to refresh your `index.html` file to see the results.
+#### [Usage](https://github.com/berinhard/pyp5js/blob/master/docs/index.md#usage)
```
Comment (berinhard) [inline README.md]: Change to https://berinhard.github.io/pyp5js/#usage

File: README.md [hunk 4]
```diff
@@ -22,53 +21,12 @@ def draw():
...
-```
-$ pyp5js monitor my_sketch
-```
-
-All of the command-line interface methods have a few optional arguments and you can check them by running:
-
-```
-$ pyp5js --help
-```
+#### [Known issues and differences to the Processing.Py and P5.js ways of doing things](known-issues-and-differences-to-the-processingpy-and-p5js-ways-of-doing-things)
```
Comment (berinhard) [inline README.md]: Same think with the other links. It should point to https://berinhard.github.io/pyp5js/#known-issues-and-differences-to-the-processingpy-and-p5js-ways-of-doing-things

File: README.md [hunk 5]
```diff
@@ -22,53 +21,12 @@ def draw():
...
-```
-
-The `index.html` is prepared to display your sketch, so you'll have to keep on opening it to refresh see results from the code you'll add to `my_sketch.py`.
-
-After updating your code, you'll have to run the `transcrypt` command to update the files. Run it as:
-
-```
-$ pyp5js transcrypt my_sketch
-```
+- [Examples](https://berinhard.github.io/pyp5js/examples/)
```
Comment (berinhard) [inline README.md]: This one is correct :+1:

**Comment by berinhard** [comment]
Ow, and another thing I've noticed: could you change the PR to be merged into `develop` instead of `master`? I'm planning a release tomorrow with the `pop` fix and also supporting the event functions. So I'd like to have the know issues sections on develop because I'll have to update it to address the event functions.

File: README.md (hunk 1 above)
Comment (villares) [reply README.md]: Done!

File: README.md (hunk 4 above)
Comment (villares) [reply README.md]: I think now I fixed it!

**Comment by berinhard** [comment]
Closing because the merge is going to master.

**Pull request closed without merging**